src/main/scala/uk/gov/nationalarchives/tdr/api/db/repository/FileRepository.scala

```diff
@@ -81,13 +92,17 @@ class FileRepository(db: Database)(implicit val executionContext: ExecutionConte
 
   def getPaginatedFiles(consignmentId: UUID,
                         limit: Int,
-                        after: Option[UUID],
+                        offset: Int,
+                        after: Option[String],
                         fileFilters: FileFilters): Future[Seq[FileRow]] = {
+    val selectedFileIds = fileFilters.selectedFileIds.getOrElse(List())
```

Comment:
I would add a new `parentId` field to the `FileFilters` case class, and use this instead of the `selectedFileIds`. This will make it more explicit what is being filtered on:

`case class FileFilters(fileTypeIdentifier: Option[String] = None, selectedFileIds: Option[List[UUID]] = None, parentId: Option[UUID] = None)`

This will also allow you to pass it directly into the `.filterOpt`:
```
...
.filterOpt(fileFilters.parentId)(_.parentid === _)
...

```